(node:993424) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-nosession-41281',
  containerIp: '10.0.3.229',
  mitmPort: 41281,
  autoAccessFixed: 1
}
2024-11-07 04:33:37.180 - [Info] MITM Version: 2
2024-11-07 04:33:37.182 - [Info] Auto Access Enabled: true
2024-11-07 04:33:37.182 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-07 04:33:37.205 - [Info] SSH man-in-the-middle server for 10.0.3.229 listening on 10.0.3.1:41281
2024-11-07 04:49:56.570 - [Debug] [Connection] Attacker connected: 47.245.81.232 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 04:49:58.052 - [Debug] [Auth] Attacker 47.245.81.232 trying to authenticate with 'password'
2024-11-07 04:49:58.053 - [Debug] [Auto Access] Attacker: 47.245.81.232, Threshold: 1, Attempts: 1
2024-11-07 04:49:58.053 - [Debug] [Auto Access] Compromising the honeypot
2024-11-07 04:49:58.053 - [Debug] [Auto Access] Adding the following credentials: 'user6:password'
2024-11-07 04:49:58.256 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-07 04:49:58.258 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-07 04:49:58.259 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.229
2024-11-07 04:49:58.329 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-07 04:49:59.367 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-07 04:50:00.911 - [Debug] [Connection] Attacker connected: 47.245.81.232 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 04:50:02.418 - [Debug] [Auth] Attacker 47.245.81.232 trying to authenticate with 'password'
2024-11-07 04:50:02.419 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-07 04:50:02.420 - [Debug] [Auth] Attacker: 47.245.81.232 has so far made 1 attempts. Remaining: 5 attempts
2024-11-07 04:50:02.953 - [Debug] [Connection] Attacker closed the connection
2024-11-07 04:50:03.180 - [Debug] [Connection] Attacker connected: 47.245.81.232 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 04:50:03.599 - [Debug] [Connection] Attacker connected: 221.222.184.230 | Client Identification: SSH-2.0-Go
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
